src/hero.css.ts

```diff
@@ -16,11 +16,11 @@ export const container = style([
     sprinkles({display: 'flex'}),
     {
         minHeight: 400,
-        height: fallbackVar(height, '100%'),
+        height,
         '@media': {
             [mq.desktopOrBigger]: {
                 minHeight: 460,
-                height: fallbackVar(height, '100%'),
+                height,
```

Comment:
I meant, if you have this already defined in base styles, there is no point in repeating it inside media query

Reply:
Yes, the fallback was redundant too